Polygon_mesh_processing/doc/Polygon_mesh_processing/Doxyfile.in

```diff
@@ -23,6 +23,7 @@ EXCLUDE_SYMBOLS += experimental
 HTML_EXTRA_FILES           = ${CGAL_PACKAGE_DOC_DIR}/fig/selfintersections.jpg \
                              ${CGAL_PACKAGE_DOC_DIR}/fig/mesh_smoothing.png \
                              ${CGAL_PACKAGE_DOC_DIR}/fig/shape_smoothing.png \
+                             ${CGAL_PACKAGE_DOC_DIR}/fig/icc_diff_radius.png
                              ${CGAL_PACKAGE_DOC_DIR}/fig/decimate_cheese.png \
```

Comment:
Missing the continuation character

Reply:
Oops.. Maybe couple of extra files Ive just added to the doc are also missing here. I will do it